cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -6,50 +6,69 @@
 
 import io.fabric8.kubernetes.api.model.ConfigMap;
 import io.fabric8.kubernetes.api.model.Container;
+import io.fabric8.kubernetes.api.model.ContainerPort;
 import io.fabric8.kubernetes.api.model.EnvVar;
 import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.PersistentVolumeClaim;
 import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.Volume;
+import io.fabric8.kubernetes.api.model.VolumeMount;
 import io.fabric8.kubernetes.api.model.extensions.Deployment;
 import io.fabric8.kubernetes.api.model.extensions.DeploymentStrategy;
 import io.fabric8.kubernetes.api.model.extensions.DeploymentStrategyBuilder;
 import io.fabric8.kubernetes.api.model.extensions.RollingUpdateDeploymentBuilder;
+import io.vertx.core.json.JsonObject;
 
 import java.util.ArrayList;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 public class KafkaConnectCluster extends AbstractModel {
+    protected static final int CLIENT_PORT = 9092;
+    protected static final String CLIENT_PORT_NAME = "clients";
 
+    protected static final int REPLICATION_PORT = 9091;
+    protected static final String REPLICATION_PORT_NAME = "replication";
     // Port configuration
     protected static final int REST_API_PORT = 8083;
     protected static final String REST_API_PORT_NAME = "rest-api";
 
     private static final String NAME_SUFFIX = "-connect";
+    private static final String METRICS_CONFIG_SUFFIX = NAME_SUFFIX + "-metrics-config";
 
     // Configuration defaults
     protected static final String DEFAULT_IMAGE =
             System.getenv().getOrDefault("STRIMZI_DEFAULT_KAFKA_CONNECT_IMAGE", "strimzi/kafka-connect:latest");
     protected static final int DEFAULT_REPLICAS = 3;
     protected static final int DEFAULT_HEALTHCHECK_DELAY = 60;
     protected static final int DEFAULT_HEALTHCHECK_TIMEOUT = 5;
+    private static final boolean DEFAULT_KAFKA_CONNECT_METRICS_ENABLED = false;
 
     // Configuration keys (in ConfigMap)
-    public static final String KEY_IMAGE = "image";
-    public static final String KEY_REPLICAS = "nodes";
+    public static final String KEY_IMAGE = "connect-image";
```

Comment:
Why have you added the `connect` prefix everywhere? While naming them with the prefix might make some sense given how they are named in Kafka config maps (which has its reasons - there are two fields always ... one for KAfka and one for ZK. Here is just Connect alone.). I wouldn't break compatibility now and would wait for the CRD implementation.